### ISSUE TITLE
Support for UA-G2-PRO

### ISSUE THREAD

**Issue by rdall96** [hjdhjd/homebridge-unifi-access#8]
I just setup the plugin in my Homebridge installation, and I can see all access hub devices fine. However, HomeKit is missing the access reader to ring for access, and I see the following line in the Homebridge logs:
```
[UniFi Access] My-Cloud-Key: UniFi Access device type 'UA-G2-PRO' is not currently supported, ignoring: UA-G2-PRO-Black-18CD [UA G2 Pro]
```

Is there a plan to support the G2 Reader Pro (and other readers with camera support) as a doorbell in HomeKit? That would allow us to view the reader as a camera in the Home app, get notified on Apple TV every time the doorbell is rang, get notifications on iPhone, and leverage HomeKit automation to perform custom actions when the doorbell is activated.

Thanks so much for all the great work you put into this!

**Comment by hjdhjd** [comment]
It already shows up as a doorbell in HomeKit. You're really asking about camera support, not doorbell support.

Let me explain:

* The UniFi Access hub is really the "doorbell" and "lock" in the Access ecosystem.
* HBUA already exposes a doorbell and a lock as a result - in fact you can even expose additional automation triggers to allow you to script things based on a doorbell ring.  **If you ring the doorbell on the G2 Pro, HBUA will trigger a doorbell ring in HomeKit**.
* What is **not** in place at the moment, and frankly may not be for quite some time, is camera support within HBUA, which would be what is needed for devices like the G2 Pro. Camera devices in the Access ecosystem use WebRTC, rather than RTSP or a direct video API like what exists in Protect. It's an entirely different game and when you dig into it it's very clear there are entirely different teams at Ubiquiti working on Access vs Protect. Supporting WebRTC can be done, but it's a bunch of work and I may or may not take it up anytime soon, but I welcome PRs, as always.

Your welcome...enjoy!

TL;DR: yes, there's already doorbell support. Your question is really about camera support. Not coming anytime soon, but welcome PRs.

**Comment by rdall96** [comment]
Oh I see, in that case yes, I was more or so asking about camera support. It's really a bummer that they use a different protocol, especially since you can pair Protect cameras with Access Hubs. I will play around more with the settings for the plugin.
I really appreciated the detailed explanation, thank you for the speedy response!
